ONBOARDING.md

```diff
@@ -14,6 +14,7 @@ invalid and are superseded by the TSC Charter.
 * Change the new member's role in the GitHub `nodejs` organization to `Owner`.
 * Add them to the `@nodejs-private` org and with an `Owner` role.
 * Add them to the `@nodejs-private/security-tsc` team.
+* Add them to the `@pkgjs` org and with an `Owner` role.
```

Comment:
it might be good to add them to a "tsc" team on the pkgjs org, so it's clear *why* they're in the org?

Reply:
@ljharb , @Trott updated.